src/AccessibilityInsights.Extensions/Interfaces/IssueReporting/IIssueReporting.cs

```diff
@@ -46,9 +46,10 @@ public interface IIssueReporting
         Task RestoreConfigurationAsync(string serializedConfig);
 
         /// <summary>
-        /// Control to let user configure/login to issue reporting service
+        /// Control to let user configure/login to issue reporting service. 
+        /// UpdateSaveButton action needs to be called when the extension is ready to save.
```

Comment:
this makes it seem like it is only called when the extension is ready to save, but is it also called when the extension is unable to save? (e.g. CanSave goes from true -> false)

I thought I heard we may be able to bind to CanSave directly (e.g. UpdateSaveButton would not be necessary) but the PR indicates it's needed. If that is true, can we consider 'NotifySaveStateChanged' or something similar?

Reply:
Binding to CanSave wasn't working 😟